node/src/chain.rs

```diff
@@ -186,8 +221,9 @@ impl ChainSrv {
     /// If register entry is read but block is not found.
     async fn load_most_recent_block<DB: database::DB>(
         db: Arc<RwLock<DB>>,
-    ) -> Result<Block> {
+    ) -> Result<(Block, Block)> {
```

Comment:
Function comment should reflect this change

Reply:
Updated.